components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1603,6 +1603,25 @@ _resolve_dependencies() {
   _validate_deps
 }
 
+function do_automatic() {
+  do_default_automatic "$1"
```

Comment:
This shouldn't need to take an argument--as is, `$1` is empty anyway. These hook functions tend to be declarative anyway.